packages/integrations/scripts/buildBundles.sh

```diff
@@ -6,7 +6,7 @@ for filepath in ./src/*; do
     # The index file is only there for the purposes of npm builds (for the CDN we create a separate bundle for each
     # integration) and the flags file is just a helper for including or not including debug logging, whose contents gets
     # incorporated into each of the individual integration bundles, so we can skip them both here.
-    if [[ $file == "index.ts" || $file == "flags.ts" ]]; then
+    if [[ $file == "index.ts" || $file == "globals.d.ts" ]]; then
```

Comment:
Given that you took out all of the package-level `globals` files, do you still need this?

Reply:
Yup you're right. Removed. f84a2157d6389b109b28b2d7bf0ee285a6bcc4d1